xain_fl/coordinator/coordinator.py

```diff
@@ -93,12 +93,12 @@ class Coordinator:  # pylint: disable=too-many-instance-attributes
 
     """
 
-    def __init__(  # pylint: disable=too-many-arguments,dangerous-default-value
+    def __init__(  # pylint: disable=too-many-arguments
         self,
         num_rounds: int = 1,
         minimum_participants_in_round: int = 1,
         fraction_of_participants: float = 1.0,
-        weights: List[ndarray] = [],  # TODO: change to non-dangerous default value
+        weights: Optional[List[ndarray]] = None,
```

Comment:
Couldn't this be removed altogether, since we're not initializing the coordinator with weights anymore?

Reply:
will be used once the the coordinator is stateless to restore crashed sessions.